Source/WebCore/PAL/pal/text/TextCodecSingleByte.cpp

```diff
@@ -174,8 +174,13 @@ static constexpr SingleByteDecodeTable ibm866 {
 template<const SingleByteDecodeTable& decodeTable> SingleByteEncodeTable tableForEncoding()
 {
     // Allocate this at runtime because building it at compile time would make the binary much larger and this is often not used.
-    // FIXME: With the C++20 version of std::count, we should be able to change this from const to constexpr and compute the size at compile time.
+    // FIXME: Remove this check after we use C++20 as a baseline
+#if __cplusplus >= 202002L
```

Comment:
Unfortunately, I don't know this is going to be enough either because of cases where one has partial support for C++20 and where other parts of the code are relying on the parts that are supported (but this case is not).

In our case, we'll probably just end up having to add a `&& !PLATFORM(PLAYSTATION)` to the check, but I don't know if this might run afoul of anyone else.